src/gui/windows/graphics.zig

```diff
@@ -24,6 +24,14 @@ const anisotropy = [_]u8{1, 2, 4, 8, 16};
 
 const resolutions = [_]u16{25, 50, 100};
 
+fn fpsCapFormatter(allocator: main.utils.NeverFailingAllocator, value: f32) []const u8 {
+	return std.fmt.allocPrint(allocator.allocator, "#ffffffFPS Limit: {d:.0}", .{value}) catch unreachable;
+}
+
+fn fpsCapCallback(newValue: f32) void {
+	settings.fpsCap = if(newValue == 144.0) null else @intFromFloat(newValue);
```

Comment:
What about 60 Hz monitors? Is there a way to get the monitor refresh rate from glfw maybe?

Reply:
60 Hz is unaffected by this line. Of course, now I realize that it prevents people from using all 144 frames of their 144 monitors :(. I think I'll just round the slider to the nearest 5, snap to 144, and unlimited.